patches/minecraft/net/minecraft/world/food/FoodData.java.patch

```diff
@@ -0,0 +1,20 @@
+--- a/net/minecraft/world/food/FoodData.java
++++ b/net/minecraft/world/food/FoodData.java
+@@ -24,9 +_,15 @@
+       this.f_38697_ = Math.min(this.f_38697_ + (float)p_38708_ * p_38709_ * 2.0F, (float)this.f_38696_);
+    }
+ 
++   @Deprecated
```

Comment:
This should have a comment/javadoc saying to use the more sensitive version in addition to the deprecated annotation.